src/components/helpers/RegMenuHelpers.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
```

Comment:
should we just define these helper functions inside of `components/Register.js`?

Reply:
i think these methods will be useful, but could we add it inside helpers/Registers.js, instead of RegMenu?

Reply:
sometimes its just convienent to not have to go to a separate file to access helper functions. 
but if you like them in a separate file then go ahead